packages/firebase_auth/firebase_auth_platform_interface/lib/src/method_channel/utils/exception.dart

```diff
@@ -25,15 +25,16 @@ Exception convertPlatformException(Object exception) {
 /// the `details` of the exception exist. Firebase returns specific codes and
 /// messages which can be converted into user friendly exceptions.
 FirebaseException platformExceptionToFirebaseAuthException(
-    PlatformException platformException) {
-  Map<String, dynamic> details = platformException.details != null
+  PlatformException platformException,
+) {
+  Map<String, dynamic>? details = platformException.details != null
       ? Map<String, dynamic>.from(platformException.details)
       : null;
 
   String code = 'unknown';
-  String /*!*/ message = platformException.message;
-  String email;
-  AuthCredential credential;
+  String? message = platformException.message;
```

Comment:
Related to the previous comment about `message` now being nullable

I believe the `/*!*/` was erroneous